components/generic/HTMLEmbed.vue

```diff
@@ -1,7 +1,19 @@
 <template>
   <!-- eslint-disable vue/no-v-html -->
   <div
-    v-if="html"
+    v-if="html && provider === 'YouTube'"
```

Comment:
Is this really only useful / needed in the case of the youtube embed, or can it be applied more generally to other embeds?